core/App/ehsm_provider.cpp

```diff
@@ -791,7 +791,7 @@ ehsm_status_t GenerateQuote(ehsm_data_t *quote)
 ehsm_status_t VerifyQuote(ehsm_data_t *quote,
                           const char *mr_signer,
                           const char *mr_enclave,
```

Comment:
title:
Return the sgx error code directly for verifyQuote

Reply:
done